pkg/secrethub/http.go

```diff
@@ -153,6 +154,13 @@ func (c *httpClient) GetMyUser() (*api.User, error) {
 	return out, errio.Error(err)
 }
 
+// SendVerificationEmail sends an email to the users registered email address for them to prove they
+// own that email address.
+func (c *httpClient) SendVerificationEmail() error {
+	rawURL := fmt.Sprintf(pathMeEmailVerification, c.base)
+	return c.get(rawURL, nil)
```

Comment:
This seems kind of weird. Shouldn't it be a `POST` call?

Reply:
Oh wow, yes this should be a POST call.